example-monorepos/blank/packages/app/package.json

```diff
@@ -8,7 +8,11 @@
     "dripsy": "^3.6.0",
     "expo-linking": "^3.0.0",
     "moti": "canary",
+    "react-native-reanimated": "2.8.0",
     "solito": "latest"
   },
+  "resolutions": {
+    "react-native-reanimated": "2.8.0"
```

Comment:
I think resolutions should go in the root `package.json`

Reply:
In the **packages/expo/package.json**  ``""react-native-reanimated": "~2.9.1"`` but the resolution is ``"react-native-reanimated": "2.8.0"``. Is there any specific reason why it's done like this ?

Reply:
I originally placed that resolution there, because on  web the app would crash with this error: software-mansion/react-native-reanimated#3355 on version 2.9.x and up. I think it might be fixed as of the latest 3.0.x version of react-native-reanimated